libraries/core/src/main/java/com/chesire/nekome/core/flags/HomeScreenOptions.kt

```diff
@@ -0,0 +1,31 @@
+package com.chesire.nekome.core.flags
+
+import android.content.Context
+import androidx.annotation.StringRes
+import com.chesire.nekome.core.R
+
+/**
+ * All possible types of a series.
+ */
+enum class HomeScreenOptions(val index: Int, @StringRes val stringId: Int) {
+    Anime(0, R.string.anime),
+    Manga(1, R.string.manga);
+
+    companion object {
```

Comment:
There should be some unit tests covering the behaviour for these methods, to match those in the `UserSeriesStatus`

Reply:
Added unit test cases following the `UserSeriesStatus` as an example.